build.js

```diff
@@ -107,7 +107,36 @@ for (const property of binaryProperties) {
 		`@unicode/unicode-${ UNICODE_VERSION }/Binary_Property/${ property }/code-points.js`
 	);
 	const set = regenerate(codePoints);
-	const output = `${ set.toCode() }\nmodule.exports = set;\n`;
+	const output = `${ set.toCode() }\nexports.characters = set;\n`;
+	fs.writeFileSync(fileName, output);
+}
+
+const propertiesOfStrings = [
+	'Basic_Emoji',
+	'Emoji_Keycap_Sequence',
+	'RGI_Emoji_Modifier_Sequence',
+	'RGI_Emoji_Flag_Sequence',
+	'RGI_Emoji_Tag_Sequence',
+	'RGI_Emoji_ZWJ_Sequence',
+	'RGI_Emoji',
+];
+for (const property of propertiesOfStrings) {
+	binaryProperties.push(property);
+
+	const fileName = `${ directory }/${ property }.js`;
+	console.log(`Creating ${ fileName }…`);
+	const rawStrings = require(`@unicode/unicode-${ UNICODE_VERSION }/Sequence_Property/${ property }/index.js`);
+	const codePoints = [];
+	const strings = [];
+	for (const rawString of rawStrings) {
+		if (rawString.length === 1 || (rawString.length === 2 && rawString.codePointAt(0) > 0xffff)) {
+			codePoints.push(rawString.codePointAt(0));
+		} else {
+			strings.push(rawString);
+		}
+	}
+	const set = regenerate(codePoints);
+	const output = `${ set.toCode() }\nexports.characters = set;\nexports.strings = ${ jsesc(strings, { minimal: true }) };\n`;
```

Comment:
I think I'd prefer not using minimal, to ensure the published code is ASCII-safe. WDYT?

```suggestion
	const output = `${ set.toCode() }\nexports.characters = set;\nexports.strings = ${ jsesc(strings) };\n`;
```

Reply:
I'm keeping `es6: true` to generate a slightly smaller output, since this package requires Node.js >= 4 anyway.

Reply:
Ack, SGTM